eacl/target.go

```diff
@@ -37,23 +37,25 @@ func (t Target) CopyTo(dst *Target) {
 	}
 }
 
-// BinaryKeys returns list of public keys to identify
+// BinaryKeys returns list of public keys or accounts to identify
 // target subject in a binary format.
 //
-// Each element of the resulting slice is a serialized compressed public key. See [elliptic.MarshalCompressed].
-// Use [neofsecdsa.PublicKey.Decode] to decode it into a type-specific structure.
+// Each element of the resulting slice is a serialized compressed public key or account from the public key.
+// See [elliptic.MarshalCompressed], [keys.PublicKey.GetScriptHash].
```

Comment:
i see u decided to set accounts as 20B script hashes instead of 25B `user.ID`. Guess @roman-khimov will be incredibly happy about this :]

i agree with the format, but if then i also want to provide proper way to work with `user.ID`s. We already have https://pkg.go.dev/github.com/nspcc-dev/neofs-sdk-go@v1.0.0-rc.12/user#ID.SetScriptHash, lets provide also provide getter

Reply:
Well, from my point of view, I thought It was the same thing. I have to revalidate these